frontend/src/routes/Governance/components/GovernanceEmptyState.tsx

```diff
@@ -77,6 +77,7 @@ export function GovernanceCreatePolicysetEmptyState(props: { rbac: boolean }) {
           components={{ bold: <strong /> }}
         />
       }
+      ignoreLoading={true}
```

Comment:
Why do we ignore loading in this case? (Probably needs a comment in the code.)

Reply:
doesn't seem to have any effect, removing